src/jit/flowgraph.cpp

```diff
@@ -22958,6 +22975,7 @@ void Compiler::fgInsertInlineeBlocks(InlineInfo* pInlineInfo)
     compLongUsed |= InlineeCompiler->compLongUsed;
     compFloatingPointUsed |= InlineeCompiler->compFloatingPointUsed;
     compLocallocUsed |= InlineeCompiler->compLocallocUsed;
+    compLocallocOptimized |= InlineeCompiler->compLocallocOptimized;
```

Comment:
Future refactoring opportunity:
We should place all of these values into a single struct so that we can perform a single struct assignment 
from InlineCompiler to the this pointer to update all of them at once.